sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -63,18 +63,23 @@ Pod::Spec.new do |spec|
   }
 
   if source[:git] then
+    hermes_utils_path = "#{react_native_path}/sdks/hermes-engine/utils"
     spec.prepare_command = <<-EOS
-    BUILD_TYPE=#{get_hermes_build_type.to_s.capitalize}
+    export BUILD_TYPE=#{ENV['PRODUCTION'] == '1' ? 'Release' : 'Debug'}
+    export RELEASE_VERSION="#{version}"
+    export IOS_DEPLOYMENT_TARGET="#{spec.deployment_target('ios')}"
+    export MAC_DEPLOYMENT_TARGET="#{spec.deployment_target('osx')}"
+    export JSI_PATH="#{react_native_path}/ReactCommon/jsi"
 
     # Set HERMES_OVERRIDE_HERMESC_PATH if pre-built HermesC is available
     #{File.exist?(import_hermesc_file) ? "export HERMES_OVERRIDE_HERMESC_PATH=#{import_hermesc_file}" : ""}
     #{File.exist?(import_hermesc_file) ? "echo \"Overriding HermesC path...\"" : ""}
 
     # Build iOS framework
-    ./utils/build-ios-framework.sh
+    #{hermes_utils_path}/build-ios-framework.sh
```

Comment:
could you double confirm the generated _Pods/Local Podspecs/hermes-engine.podspec.json_ doesn't contain the absolute path? in my case, it will have `/Users/kudo`, e.g.
`  "prepare_command": "export BUILD_TYPE=Debug\nexport RELEASE_VERSION=\"1000.0.0\"\nexport IOS_DEPLOYMENT_TARGET=\"12.4\"\nexport MAC_DEPLOYMENT_TARGET=\"10.13\"\nexport JSI_PATH=\"/Users/kudo/react-native/sdks/hermes/../../ReactCommon/jsi\"\n\n# Set HERMES_OVERRIDE_HERMESC_PATH if pre-built HermesC is available\n\n\n\n# Build iOS framework\n/Users/kudo/react-native/sdks/hermes/../../sdks/hermes-engine/utils/build-ios-framework.sh\n\n# Build Mac framework\n/Users/kudo/react-native/sdks/hermes/../../sdks/hermes-engine/utils/build-mac-framework.sh"`

that would make the checksum in Podfile.lock unstable between each environments. that's why previously i proposed environment variables approach.

Reply:
You are right! I have full path too... I'm updating the PR to use the env variable.

Reply:
it looks good now, thanks @cipolleschi!